icon_validator/workflow/model.py

```diff
@@ -1,65 +1,167 @@
 from typing import List, Optional, Type, Union
-from dataclasses import dataclass, field
+from dataclasses import field
+from pydantic import BaseModel
 
 
 # Kom/Icon file
-@dataclass
-class TriggersInputActorProperties:
-    assets: dict = field(default_factory=lambda: {})
-    users: dict = field(default_factory=lambda: {})
+
+class TriggersOutputJsonSchemaPropertiesType(BaseModel):
```

Comment:
this is a nitpick but these class names are really long! Is the "JsonSchema" that is repeated across most of them needed?

Reply:
Good point! Wanted to try and differentiate between some overlapping attribute names but should make this more readable.